roles/pbench_agent_config/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# pbench agent configuration
+- name: "pbench agent configuration - install config file(s)"
+  ansible.builtin.include_role:
+    name: pbench_agent_files_install
+  vars:
+    source: "{{ pbench_config_url }}"
```

Comment:
Where/how is `pbench_config_url` defined, and what does it get defined to?  (Is it going to pull a file from the pbench Git repo?...if os, which _branch_ is it going to reference?)